parquet-column/src/main/java/org/apache/parquet/column/values/boundedint/DevNullValuesWriter.java

```diff
@@ -40,6 +40,11 @@ public void reset() {
   }
 
   @Override
+  public void close() {
+
+  }
```

Comment:
As a decent number of these have actual content, I think it might be better to leave the interface abstract to force subclass authors think about the resources they are using.